component/s3storage/client.go

```diff
@@ -128,6 +129,9 @@ func (cl *Client) Configure(cfg Config) error {
 		config.WithCredentialsProvider(credentialsProvider),
 		config.WithAppID(UserAgent()),
 		config.WithRegion(cl.Config.authConfig.Region),
+		config.WithRetryer(func() aws.Retryer {
```

Comment:
Yes this was mainly to speed up testing, and I think we've been slowly increasing it as we do more testing because some operations end up taking a while to get to the cloud. We could probably return it back to normal, but it can take up to 20-30 seconds on an operation with full attempts and timeout